test/Microsoft.AspNetCore.Http.Tests/HttpContextFactoryTests.cs

```diff
@@ -35,7 +35,7 @@ public void AllowsCreatingContextWithoutSettingAccessor()
             var context = contextFactory.Create(new FeatureCollection());
             contextFactory.Dispose(context);
         }
-
+/*
```

Comment:
Try this:
```c#
var setupInfo = new AppDomainSetup
{
   ApplicationBase = AppDomain.CurrentDomain.BaseDirectory
};
var newDomain = AppDomain.CreateDomain("new", null, setupInfo);
```
cref https://github.com/Microsoft/vstest/issues/419#issuecomment-276834482